src/Network/Network/help/New-AzExpressRoutePortIdentity.md

```diff
@@ -0,0 +1,114 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azexpressrouteportidentity
+schema: 2.0.0
+---
+
+# New-AzExpressRoutePortIdentity
+
+## SYNOPSIS
+Creates an Azure ExpressRoutePortIdentity.
+
+## SYNTAX
+
+```
+New-AzExpressRoutePortIdentity -UserAssignedIdentityId <String> [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
```

Comment:
Cmdlet doesn't support `ShouldProcess`, please remove `WhatIf` and `Confirm` from readme.

Reply:
Done